src/test/SeleniumWebDriverTest.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SeleniumWebDriverTest {
+    @Test
+    public void testDriver() throws InterruptedException {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
```

Comment:
Seems, you forgot to add the driver file. This test should not work

Reply:
Do you mean I need to add chromedriver.exe too?